CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPlugin.java

```diff
@@ -186,7 +184,7 @@ public PluginParameters createParameters() {
         final PluginParameter<StringParameterValue> string1 = StringParameterType.build(TEST1_PARAMETER_ID);
         string1.setName("Test1");
         string1.setDescription("A test string");
-        string1.setStringValue("Plugh.");
+//        string1.setStringValue("Plugh.");
```

Comment:
There are lots of these bits of code commented. Is there any reason for this or should they be uncommented?